linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/bash/ubuntu.sh

```diff
@@ -1,8 +1,6 @@
 # platform = multi_platform_ubuntu
 
-if command -v nmcli >/dev/null 2>&1 ; then
-    nmcli radio all off
```

Comment:
please keep the nmcli, this aligns with CIS

Reply:
As discussed, we can remove this since CIS 2.0.0 no longer uses nmcli in the remediation.